bucket/olive-editor-nightly.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": "5ef56c80",
+    "description": "Non-linear video editor aiming to provide a fully-featured alternative to high-end professional video editing software.",
+    "homepage": "https://olivevideoeditor.org",
+    "license": "GPL-3.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://nightly.link/olive-editor/olive/workflows/ci/master/Olive-5ef56c80-Windows-x86_64.zip.zip",
+            "hash": "f79d3816c1b9a13fa35c17f087d9bcea2b8f29febdc0e74acc5587e8e203b8a8"
+        }
+    },
+    "pre_install": [
+        "7z x \"$dir\\Olive*.zip\" -o\"$dir\\tmp\"",
+        "Move-Item -Path \"$dir\\tmp\\olive-editor\\*\" -Destination \"$dir\\\"",
+        "Remove-Item \"$dir\\tmp\" -Recurse",
+        "'config.xml', 'layout', 'shortcuts' | ForEach-Object { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
```

Comment:
```
---------------------------
Error loading settings
---------------------------
Failed to load application settings. This session will use defaults.

Premature end of document.
---------------------------
OK   
---------------------------
```
Getting this error, maybe you can fill in config.xml with an example setting?

Reply:
Ok done.